README.md

```diff
@@ -92,43 +94,321 @@ It even works on MacOSX in addition to the OS scroll acceleration,
 such that you can further increase the acceleration.
 
 
-## Dependencies
+## Prerequisites
+
+### `pipx`
+
+`pipx` is recommended to install the application so that it does not interfere
+with other packages or environments.
+It works by creating a virtual environment for the application,
+installing the application into it, and then making any executables available
+in the system path.
+
+To install `pipx`, see [here](https://pipx.pypa.io/stable/installation/).
+
 
-    pip install -r requirements.txt
+## Installation
+
+There are a few ways to install the application.
```

Comment:
You list the three options: pipx, venv+pip, system Python + pip. Shouldn't you also list uv?

Reply:
I'm not sure, I read a lot of times that uv is also quite popular just for users, not just for developers.

So I would also include it here. And anyway, stay neutral, don't write about recommendations here (because I think those are very subjective - I actually think more people would recommend uv, not pipx). Maybe link one or two good objective articles on this subject / such comparison.

Reply:
E.g., there is `uv tool install <package>`, and `uvx <command>` (or `uv tool run <command>`).